### PR TITLE
feat(ChoiceGroup): increase rendering flexibility

### PR DESCRIPTION
**Pull request by silvenon**
By accepting a function as `children` we're increasing the flexibility of rendering, for example now people can combine `react-window` with ChoiceGroup.

- [Slack request](https://skypicker.slack.com/archives/C9B1H0ACW/p1622462413007500)

 Storybook: https://orbit-feat-choice-group-flexibility.surge.sh

### CODE REVIEW

**Comment by silvenon** [comment]
The spacing between items in the "Render prop" story is off, but it's fine to review otherwise.

**Comment by silvenon** [comment]
Fixes spacing with some styling gymnastics. 😄 ✅

**Comment by mainframev** [comment]
this PR creates empty file in ` docs/plugins/table-of-contents/gatsby-node.js ` Seems like not related to this PR.

File: packages/orbit-components/src/ChoiceGroup/ChoiceGroup.stories.js [hunk 1]
```diff
@@ -93,6 +94,56 @@ WithError.story = {
   },
 };
 
+export const RenderProp = (): React.Node => {
+  const boxShadowSize = "3px";
+  return (
+    <ChoiceGroup label="What was the reason for your cancellation?" onChange={() => {}}>
```
Comment (mainframev) [inline packages/orbit-components/src/ChoiceGroup/ChoiceGroup.stories.js:100]:
```suggestion
    <ChoiceGroup label="What was the reason for your cancellation?" onChange={action("onChange")}>
```

**Review by mainframev — CHANGES_REQUESTED** [review]

**Review by vepor — APPROVED** [review]

**Pull request merged**